pkgdiff.pl

```diff
@@ -54,14 +54,15 @@
 use Config;
 use Fcntl;
 
-my $TOOL_VERSION = "1.8";
```

Comment:
It's already bumped. Next release is `1.8`.

Reply:
OK :+1: